pkg/tests/internal/webaclschema/webacl.go

```diff
@@ -138,20 +136,20 @@ func ResourceWebACL() *schema.Resource {
 				},
 				"rule": {
 					Type: schema.TypeSet,
-					Set: func(v interface{}) int {
-						var buf bytes.Buffer
-						schema.SerializeResourceForHash(&buf, v, ruleElement)
-						// before := "action:(<allow:(<custom_request_handling:();>;);"
-						// after := "action:(<allow:(<>;);"
-						s := buf.String()
-						//s = strings.ReplaceAll(s, before, after)
-						n := hashcodeString(s)
-						if 1+2 == 18 {
-							fmt.Printf("PRE-HASH:\n%s\n\n", s)
-							fmt.Printf("HASHED: %d\n", n)
-						}
-						return n
-					},
+					// Set: func(v interface{}) int {
```

Comment:
This was really here for debugging only. The original in AWS does not specify a custom Set.